### PR TITLE
update in-no config

### PR DESCRIPTION
**Pull request by mathilde-daugy**
## Description

Update unknown emission factor as the unknown share of production is mainly biomass and hydro. The EF calculation was done using CEA values for 2022
[IN unknown emission calculation ](https://docs.google.com/spreadsheets/d/1on86UoTcljjLztobbPWMMXZZvXON12uKAGIH1ftTypw/edit#gid=0)

### Preview

NA

### CODE REVIEW

File: config/zones/IN-NO.yaml [hunk 1]
```diff
@@ -20,7 +20,17 @@ contributors:
...
 timezone: Asia/Kolkata
+emissionFactors:
+  direct:
+    unknown:
+      source: CEA 2022; assumes 56% biomass, 36% hydro, 8% unknown
+      value: 45
+  lifecycle:
+    unknown:
+      source: CEA 2022; assumes 56% biomass, 36% hydro, 8% unknown
+      value: 192
```
Comment (VIKTORVAV99) [inline config/zones/IN-NO.yaml:33]: We should also add isRenewable and isLowCarbon values as well so we get the "correct" renewable and low carbon share.

**Review by VIKTORVAV99 — CHANGES_REQUESTED** [review]
We just need to add isRenewabe and isLowCarbon overrides for unknown as well.

File: config/zones/IN-NO.yaml [hunk 2]
```diff
@@ -20,7 +20,23 @@ contributors:
...
+isLowCarbon:
+  unknown:
+    value: 1
```
Comment (VIKTORVAV99) [inline config/zones/IN-NO.yaml:36]:
```suggestion
    value: 0.92
```
Shouldn't it be the sum of known renewable of the unknown?

File: config/zones/IN-NO.yaml [hunk 3]
```diff
@@ -20,7 +20,23 @@ contributors:
...
+isRenewable:
+  unknown:
+    value: 1
```
Comment (VIKTORVAV99) [inline config/zones/IN-NO.yaml:39]:
```suggestion
    value: 0.92
```

**Review by VIKTORVAV99 — APPROVED** [review]
I have a suggestion but otherwise things look good! 👍🏼

File: config/zones/IN-NO.yaml (hunk 2 above)
Comment (mathilde-daugy) [reply config/zones/IN-NO.yaml:36]: i guess but the remainder is also other renewables according to the data source so I put 1 
I think we can put 0.92 to be on the safe side

Comment (VIKTORVAV99) [reply config/zones/IN-NO.yaml:36]: Ah in that case I leave it up to you 👍🏼

Comment (mathilde-daugy) [reply config/zones/IN-NO.yaml:36]: i'll put 0.92 as we have no idea what the 8% are and are using the default unknown EF

**Pull request merged**